spec/features/user_invites_friend_spec.rb

```diff
@@ -31,6 +31,10 @@ def friend_signs_up_from_invite
     expect(page).to have_xpath("//input[@value='friend@friend.com']")
     fill_in "Password", with: "password"
     fill_in "Full Name", with: "Mr Friend"
+    fill_in "Credit Card Number", with: "4242424242424242"
+    fill_in "Security Code", with: "123"
+    select "1 - January", from: "date_month"
+    select "2017", from: "date_year"
     click_button "Sign Up"
```

Comment:
There is actually 1 hidden thing here. After registration, since you are using Stripe - stripe.js issues an Ajax request to stripe server with card data and returns token. The problem here is your specs might fail, because capybara does not wait for the ajax to resolve. Therefore, you might have to put artificial sleep here like `sleep 3` or depending on how good your connection is.